deployment/helm/balloons/values.yaml

```diff
@@ -28,6 +28,8 @@ config:
     klog:
       skip_headers: true
   instrumentation:
+    httpEndpoint: ":8891"
+    prometheusExport: true
```

Comment:
Adding this setting will turn on metrics exposing by default. Is that your intention?

Reply:
Thanks for spotting. Probably no reason to change the current defaults in that regard. Changed to `false`.

Another thing I realized is that maybe we always want to unconditionally annotate the pod for autodiscovery, even if we don't start with prometheus metrics exporting enabled. Since we can
can dynamically change the configuration to enable it any time.